src/adapter/src/coord.rs

```diff
@@ -381,15 +381,25 @@ pub struct Coordinator<S> {
     /// active connections.
     active_conns: HashMap<ConnectionId, ConnMeta>,
 
-    /// For each identifier, its read policy and any transaction holds on time.
+    /// For each identifier in STORAGE, its read policy and any read holds on time.
     ///
     /// Transactions should introduce and remove constraints through the methods
     /// `acquire_read_holds` and `release_read_holds`, respectively. The base
     /// policy can also be updated, though one should be sure to communicate this
     /// to the controller for it to have an effect.
     ///
     /// Access to this field should be restricted to methods in the [`read_policy`] API.
-    read_capability: HashMap<GlobalId, ReadCapability<mz_repr::Timestamp>>,
+    storage_read_capability: HashMap<GlobalId, ReadCapability<mz_repr::Timestamp>>,
+    /// For each identifier in COMPUTE, its read policy and any read holds on time.
+    ///
+    /// Transactions should introduce and remove constraints through the methods
+    /// `acquire_read_holds` and `release_read_holds`, respectively. The base
+    /// policy can also be updated, though one should be sure to communicate this
+    /// to the controller for it to have an effect.
+    ///
+    /// Access to this field should be restricted to methods in the [`read_policy`] API.
+    compute_read_capability: HashMap<GlobalId, ReadCapability<mz_repr::Timestamp>>,
```

Comment:
Nit, but since we are touching this code anyway: Shouldn't these field names be plural, i.e. `storage_read_capabilities` and `compute_read_capabilities`?

Reply:
Done